src/ResourceManager/Network/Commands.Network/help/New-AzureRmIpsecPolicy.md

```diff
@@ -0,0 +1,172 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+online version: 
+schema: 2.0.0
+---
+
+# New-AzureRmIpsecPolicy
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
+
+## SYNTAX
+
+```
+New-AzureRmIpsecPolicy -SALifeTimeSeconds <Int32> -SADataSizeKilobytes <Int32> -IpsecEncryption <String>
+ -IpsecIntegrity <String> -IkeEncryption <String> -IkeIntegrity <String> -DhGroup <String> -PfsGroup <String>
+```
+
+## DESCRIPTION
+{{Fill in the Description}}
```

Comment:
@henry416 there are a few templates left behind from `platyPS` when generating the markdown for the first time. Would you mind replacing these templates (description, examples, etc.) with useful information for the user?